main.cpp

```diff
@@ -117,6 +132,76 @@ Thing 4) corgi
     3) play catch
  */
 
+ /*
+Thing 5) VR: External camera
+5 properties:
+    1) amount of megapixels on camera
+    2) camera diameter in mm
+    3) camera resolution
+    4) camera color mode
+    5) camera aperture
```

Comment:
your Thing's name includes the word `camera`.
None of these properties need to include that word.
- [ ] revise